src/gluonnlp/models/albert.py

```diff
@@ -314,24 +422,37 @@ def get_initial_embedding(self, F, inputs, token_types=None):
         ----------
         F
         inputs
-            Shape (batch_size, seq_length)
+            - layout = 'NT'
+                Shape (batch_size, seq_length)
+            - layout = 'TN'
+                Shape (seq_length, batch_size)
         token_types
-            Shape (batch_size, seq_length)
+            - layout = 'NT'
+                Shape (batch_size, seq_length)
+            - layout = 'TN'
             If None, it will be initialized as all zero
 
         Returns
         -------
         embedding
             The initial embedding that will be fed into the encoder
+            - layout = 'NT'
+                Shape (batch_size, seq_length, C_embed)
+            - layout = 'TN'
+                Shape (seq_length, batch_size, C_embed)
         """
+        if self.layout == 'NT':
+            time_axis, batch_axis = 1, 0
```

Comment:
would this be more intuitive by swapping the order like `batch_axis, time_axis = 0, 1`

Reply:
solved